src/simmate/toolkit/structure_prediction/evolution/database/evolutionary_search.py

```diff
@@ -382,8 +395,16 @@ def individuals_datatable(self):
     def individuals(self):
         # note we don't call "all()" on this queryset yet becuase this property
         # it often used as a "base" queryset (and additional filters are added)
+        composition = Composition(self.composition)
         return self.individuals_datatable.objects.filter(
-            formula_full=self.composition,
+            # You'd expect this filter to be...
+            #   formula_full=self.composition
+            # However, this misses structures that are reduced to a smaller
+            # unitcells during relaxation. Therefore, by default, we need to
+            # include all individuals that have fewer nsites. This is
+            # often what a user wants anyways during searches, so it works out.
+            formula_reduced=composition.reduced_formula,
+            nsites__lte=composition.num_atoms,
```

Comment:
@scott-materials

I found a big mistake/bug in the search algorithm. "Individuals" were originally filtered based on the full formula, which made sense because we're doing a fixed-composition search. I only ever tested fully-reduced compositions, so this worked just fine. However, for compositions like Ca8N4, any structures that experience a drop in symmetry (e.g. reduced to a Ca4N2 unitcell) were being thrown out from the search.

To fix this, I now define individuals to be all structures with equal or fewer atoms + the same reduced formula. "fixed-composition" still means that **new** structures from creators and transformations will always be the desired nsites, but results will include smaller unitcells as well.